streetcrm/mixins.py

```diff
@@ -134,10 +134,9 @@ def get_urls(self, *args, **kwargs):
         )
         return urls
 
-    # This is based on STREETCRMAdminSite.export_search, but was different enough
```

Comment:
Well, you get rid of the comment about this method's similarity with STREETCRMAdminSite.export_search, but now the method is very similar to EventAdmin.export_attendees (in admin.py)... Is that similarity worth remarking on in both places?  (Or is there even some further abstraction that could be done?)

Reply:
Well, I guess I considered the duplication there to be more configuration duplication rather than logic duplication.  I think that kind of stuff should stay duplicated so that it creates a difference between what the code expects you to customize, and what it doesn't.  Happy to be overridden.

(the comment was added from when I moved the logic code to the StreetCRMAdminSite part of the codebase)